anyway/infographics_utils.py

```diff
@@ -1286,8 +1236,12 @@ def get_latest_accident_date(table_obj, filters):
 def generate_widgets(request_params: RequestParams, to_cache: bool = True) -> List[Widget]:
```

Comment:
Does this function has side effect (through `create_widget()`) other than returning the widgets? If so, I think it we need to document it.

Reply:
It does not. I instantiates the widgets and returns them.